content/docs/command-reference/metrics/diff.md

```diff
@@ -80,7 +72,7 @@ Start by creating a simple metrics file and commit it:
 
 ```dvc
 $ dvc run -M metrics.json \
-        'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
+          'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
```

Comment:
On running this command in dvc version 1.0.0a6:

I get the following error:
error: `-n | --name'

Reply:
The examples weren't ready yet. Please try the latest version and lmk.